### PR TITLE
(1/2)review and fix /src/_locales/ja/messages.json

### PR DESCRIPTION
**Pull request by blank71**
[Task] Japanese l10n Copywriting Review https://github.com/DimensionDev/Maskbook/issues/1569

### CODE REVIEW

File: src/_locales/ja/messages.json [hunk 1]
```diff
@@ -140,7 +140,7 @@
     "person_or_group_in_list_many": "{{people}}… 全部で{{count}}",
     "person_or_group_in_list_many_no_preview": "全部で{{count}}",
     "personas": "人格",
-    "please_include_proof_your_bio": "以下の「 」内の内容を自己紹介欄に追加すれば Maskbook が使えるようになります。「{{bio}}」",
+    "please_include_proof_your_bio": "「{{bio}}」の中身をコピーし、プロフィールの自己紹介欄に追加してしてください。でなければ Maskbook を正常に使用できない場合があります。",
     "popup_current_persona": "現在の人格アカウント",
     "popup_current_wallet": "",
```
Comment (septs) [inline src/_locales/ja/messages.json:145]: the is empty.

File: src/_locales/ja/messages.json [hunk 2]
```diff
@@ -265,64 +265,64 @@
     "wallet_select_wallet": "財布",
     "wallet_confirmation_hint": "仮想通貨ウォレットを作成する必要があることを確認します。",
     "wallet_new": "",
```
Comment (septs) [inline src/_locales/ja/messages.json:267]: the is empty.

**Review by septs — CHANGES_REQUESTED** [review]

File: src/_locales/ja/messages.json (hunk 1 above)
Comment (blank71) [reply src/_locales/ja/messages.json:145]: Sorry what does "the is empty." mean?

Comment (septs) [reply src/_locales/ja/messages.json:145]: Please help us fill in the content, thank

Comment (septs) [reply src/_locales/ja/messages.json:145]: Please search `""` string.

**Comment by septs** [comment]
waiting for @neruthes and @Tedko  review

**Review by Tedko — APPROVED** [review]

**Comment by Tedko** [comment]
Look good to me

**Comment by Tedko** [comment]
@blank71 many thx!

File: src/_locales/ja/messages.json [hunk 3]
```diff
@@ -260,69 +260,69 @@
...
+    "plugin_red_packet_description_incoming": "開封可能です",
+    "plugin_red_packet_description_fail": "作成失敗",
+    "plugin_red_packet_description_claimed": "既に受け取り済みです",
+    "plugin_red_packet_description_expired": "期限切れです",
+    "plugin_red_packet_description_claim_pending": "受け取り中",
+    "plugin_red_packet_description_empty": "期限切れです",
+    "plugin_red_packet_description_failover": "合計：{{total}} {{name}} / {{shares}} 株",
     "plugin_gitcoin_display_name": "プラグイン: Gitcoin Grant",
-    "plugin_gitcoin_readme": "このサービスを利用するで、寄附金額の 5% は <fund>Gitcoin grants development fund</fund> へ寄附しています。",
+    "plugin_gitcoin_readme": "このサービスを利用するにあたり、寄付金額の 5% が <fund>Gitcoin grants development fund</fund> に寄付されます",
```
Comment (septs) [inline src/_locales/ja/messages.json:303]:
```suggestion
    "plugin_gitcoin_readme": "このサービスを利用するにあたり、寄付金額の 5% が <fund>Gitcoin grants development fund</fund> に寄付されます",
```

`Gitcoin grants development fund` not translated to Japanese?

**Review by Tedko — APPROVED** [review]

File: src/_locales/ja/messages.json (hunk 3 above)
Comment (Tedko) [reply src/_locales/ja/messages.json:303]: I think we don't need to translate that.

Gitcoin itself don't have a Japanese version. Leave it as it

Comment (septs) [reply src/_locales/ja/messages.json:303]: OK.

File: src/_locales/ja/messages.json [hunk 4]
```diff
@@ -260,69 +260,69 @@
...
-    "plugin_file_service_show_more": "他にも",
-    "plugin_file_service_file_key": "ファイルキー:",
+    "plugin_file_service_show_more": "更に",
+    "plugin_file_service_file_key": "File Key:",
     "plugin_file_service_unencrypted": "このファイルは暗号化されていません",
     "plugin_file_service_signing_failed": "サービスが使用できません。今しばらくお待ちください。",
-    "plugin_file_service_on_encrypt_it": "暗号化しよう",
+    "plugin_file_service_on_encrypt_it": "暗号化しましょう",
     "plugin_file_service_on_show_more": "サポートされていません",
     "plugin_file_service_legal_text": "このプラグインを使用することで<terms>利用規約</terms>並びに<policy>個人情報保護方針（プライバシーポリシー）</policy>に同意したこととなります。",
```
Comment (septs) [inline src/_locales/ja/messages.json:327]:
```suggestion
    "plugin_file_service_legal_text": "このプラグインを使用することで<terms>利用規約</terms>並びに<policy>個人情報保護方針（プライバシーポリシー）</policy>に同意したこととなります。",
```

`<terms>利用規約</terms>` and `<policy>個人情報保護方針（プライバシーポリシー）</policy>`

@Tedko keep original text?

https://legal.maskbook.com/arweave/file-service/

File Service not include other (non English) language legal file.

File: src/_locales/ja/messages.json (hunk 3 above)
Comment (blank71) [reply src/_locales/ja/messages.json:303]: I thought it is a name of some organization. Isn't it?
If yes, remain it is fine for me.

File: src/_locales/ja/messages.json (hunk 4 above)
Comment (septs) [reply src/_locales/ja/messages.json:327]: CC @Tedko @neruthes

Comment (Tedko) [reply src/_locales/ja/messages.json:327]: Both work

Comment (septs) [reply src/_locales/ja/messages.json:327]: <ruby>Privacy<rt>プライバシー</rt> Policy<rt>ポリシー</rt></ruby>

**Review by septs — DISMISSED** [review]

**Review by neruthes — DISMISSED** [review]

**Comment by septs** [comment]
Because we merged Poll plugin (https://github.com/DimensionDev/Maskbook/pull/1578)

There are some conflicts, please handle.

@blank71 

thanks.

**Comment by blank71** [comment]
I don't know how to use Git/GitHub well so... am I correct?

**Comment by blank71** [comment]
Resolved conflict and make PR again is the right way?

**Review by Tedko — DISMISSED** [review]

**Comment by Tedko** [comment]
@septs take a look. Should be ok. If ok pls merge

File: src/_locales/ja/messages.json [hunk 5]
```diff
@@ -344,4 +344,19 @@
     "plugin_trader_table_volume": "取引高（24時間）",
     "plugin_trader_settings_data_source_primary": "データーのソース",
     "plugin_trader_settings_data_source_secondary": "異なるプラットフォームからトレンドデーターを取得"
```
Comment (septs) [inline src/_locales/ja/messages.json]:
```suggestion
    "plugin_trader_settings_data_source_secondary": "異なるプラットフォームからトレンドデーターを取得",
```

**Pull request merged**